### ISSUE TITLE
Results on PASCAL, Cityscapes ?

### ISSUE THREAD

**Issue by mhashas** [petko-nikolov/semseg.pytorch#2]
I am wondering what results did you obtain on datasets such as PASCAL or cityscapes ?

**Comment by petko-nikolov** [comment]
Hey,

Please find below some of the models I trained and was able to achieve an accuracy as stated in their respective paper.

| Model               | Dataset    | mIOU |
|---------------------|------------|------|
| VGG FCN8            | Pascal VOC | 62.2 |
| DeepLabv3 Resnet101 | Pascal VOC | 72.9 |
| FCDenseNet67        | CamVid     | 65.5 |

I haven't trained these models on Cityscapes yet, but that's about to be done soon. Also, I am about to publish some pre-trained models on Pascal VOC (hopefully Cityscapes as well) within the next few weeks.
